Dockerfile.amd64

```diff
@@ -22,9 +22,10 @@ LABEL maintainer "Casey Davenport <casey@tigera.io>"
 ARG ARCH=amd64
 
 # Install a backported version of iptables to ensure we have version 1.6.2
+# Similarly for iproute2, we want a more recent version for BPF support.
 RUN printf "deb http://deb.debian.org/debian stretch-backports main\n" > /etc/apt/sources.list.d/backports.list \ 
     && apt-get update \
-    && apt-get -t stretch-backports install -y iptables 
+    && apt-get -t stretch-backports install -y iptables iproute2
```

Comment:
Any idea how our CI missed this?

How can we be confident that we don't need down versions of other deps?